apps/web/components/CheckoutAddress/CheckoutAddress.vue

```diff
@@ -62,31 +111,48 @@ const props = withDefaults(defineProps<CheckoutAddressProps>(), {
 });
 const { data: cart } = useCart();
 const editMode = ref(false);
+const { data: addresses, setDefault, deleteAddress, defaultAddressId } = useAddress(props.type);
 
 const cartAddress = computed(() =>
   props.type === AddressType.Billing
     ? cartGetters.getCustomerInvoiceAddressId(cart.value)
     : cartGetters.getCustomerShippingAddressId(cart.value),
 );
 
-const selectedAddress = computed(
+const defaultAddress = computed(
```

Comment:
it looks like we could simplify into something like:

```
const selectedAddress = ref(
  props.addresses.find((address) => userAddressGetters.getId(address) === cartAddress?.value?.toString()) ??
    ({} as Address),
);
```

Reply:
good idea, but how do i change the selected address change action, when:

const setNewSelectedAddress = (selectedAddressNew: Address) => {
  selectedAddress.value = selectedAddressNew;
};

is missing as well on line 130 -132. @csandru-plenty

Reply:
<img width="884" alt="Screenshot 2024-04-25 at 13 45 20" src="https://github.com/plentymarkets/plentyshop-pwa/assets/149154151/c7c64deb-e0a6-4dba-95dc-39d92b1dc47a">

Reply:
it should be fine this way, or did I misunderstood you?

Reply:
yes works like this but i thought i should replace this as well, since its included in line 122 - 133. Then i will push it like this. @csandru-plenty